playfab-docs/features/multiplayer/servers/determining-required-dlls.md

```diff
@@ -27,6 +27,10 @@ Use the [ListDlls](https://docs.microsoft.com/sysinternals/downloads/listdlls) u
 ## Compare required DLLs with those included in container
 With the list of system DLLs determined, the next step is to determine which of these system DLLs are already included in the container.
 1. Download the PlayFab multiplayer container image. To download the container image, run the setup.ps1 script referenced in [local debug steps](locally-debugging-game-servers-and-integration-with-playfab.md). Alternatively, allocate a new MPS instance and connect via Remote Desktop. All Windows MPS instances will have the container image downloaded.
-1. Start a command line session in a new docker container by running the command `docker run -it mcr.microsoft.com/playfab/multiplayer:wsc-10.0.17134.950 cmd` in an administrator command window. Replace "wsc-10.0.17134.950" with the tag of the docker image you downloaded in step 1. If you are not sure of the correct tag, run `docker images` to list the downloaded images. ![docker images output](media/docker-images.png)
+1. Start a command line session in a new docker container by running the following command. You might need to run an administrator command window.
+```bash
```

Comment:
Why does it say bash? People might just be using regular powershell for this right?

Reply:
Yeah, “bash” string itself is not visible, it’s only for syntax highlighting 

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code